### ISSUE TITLE
DataAccessException triggered when handling a large number of CBZ files

### ISSUE THREAD

**Issue by Kuan-Lun** [gotson/komga#1669]
### Steps to reproduce

1. Place 2,788 CBZ files, totaling 59 GB, in a single directory.
2. Add the directory to libraries, unchecking all settings.
3. Click on "Scan library files" (the scan does not start automatically after adding).
4. A certain number of CBZ files will be scanned, and the scanned files will appear as "To be analyzed" in Komga.
5. The scan interrupts, and an error is displayed in the logs (see Logs).
*. Repeating steps 3 to 5 multiple times eventually allows the entire library to be scanned.

### Expected behavior

The scan should complete after a single click of the scan button.

### Actual behavior

Please refer to the Steps to reproduce.

### Logs

[mylogs.csv](https://github.com/user-attachments/files/16775567/mylogs.csv)

### Komga version

v1.11.3-master

### Operating system

Linux

### Installation method

Docker

### Other details

No.

### Acknowledgements

- [X] I have searched the existing issues (open AND closed) and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/docs/faq).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
your database is not reachable or too slow:

```
SqliteMainPool - Connection is not available, request timed out after 30000ms
```

**Comment by Kuan-Lun** [comment]
I understand. However, could this possibly be due to a one-time insertion of too much data into the "local" database?

**Comment by gotson** [comment]
no, Komga processes data sequentially, people have been successfully processing TBs of data in Komga without any problem. Most likely the issue lies in how you host Komga.

Can you share your dockerfile, and provide details on how the drives are mapped, and what is the underlying hardware and storage infrastructure ?

**Comment by Kuan-Lun** [comment]
I understand that Komga can handle TBs of data without issues, as I successfully processed 2 TB of data with the older version ([v1.10.4-master](https://github.com/gotson/komga/issues/1474)) without any problems. I believe this issue might be related to the new version of Komga.

Dockerfile: [docker-compose.yml](https://gofile.io/d/uRoiiy)
Drives: Synology DS1520+

**Comment by gotson** [comment]
Nothing has changed on that front for quite some time. On Synology you may need to check that the config folder is not indexed / managed by Synology in any way. Syno media indexing can be very taxing on some folders.

**Comment by Kuan-Lun** [comment]
> I have the same issue but on ubuntu with komga, version 1.14.1, via docker.
> 
> Did you find a solution for this?

No.
